lib/core/src/Cardano/Wallet/Primitive/Types/Tx.hs

```diff
@@ -537,8 +538,8 @@ data SealedTx = SealedTx
     -- always be True, but it will be False if 'mockSealedTx' is used to
     -- construct a 'SealedTx' for unit tests.
 
-    , cardanoTx :: InAnyCardanoEra Cardano.Tx
-    -- ^ Decoded transaction.
+    , unsafeCardanoTx :: InAnyCardanoEra Cardano.Tx
```

Comment:
We should no be able to re-write this as `variants :: NonEmpty (InAnyCardanoEra Cardano.Tx)`, but I didn't do it now.